### PR TITLE
Multiple to Single Action Sound Effect

### PR DESCRIPTION
**Pull request by talios0**
Instead of allowing an array of sound effects for each action, the character is now limited to a single sfx.

The state sfx are still arrays, allowing for multiple sound effects to be included.

The test character was also modified to accommodate this change.

### CODE REVIEW

**Review by Apexal — APPROVED** [review]
LGTM!

File: src/interfaces/action.ts [hunk 1]
```diff
@@ -28,5 +28,5 @@ export interface Action {
   /** The ordered frames of the animation */
   animation: AnimationFrame[];
   /** Possible sound effects for the action. One is chosen randomly each time the move is performed. */
-  soundEffects: SoundEffect[];
+  soundEffects: SoundEffect;
```
Comment (Apexal) [inline src/interfaces/action.ts:31]: This property name should be changed to `soundEffect` and the type should be nullable (`| null`).

**Review by Apexal — CHANGES_REQUESTED** [review]

File: src/interfaces/action.ts (hunk 1 above)
Comment (talios0) [reply src/interfaces/action.ts]: Good point!

**Comment by talios0** [comment]
Just changed the "soundEffects" property from within action to be called "soundEffect" and made it nullable. I have also gone and updated the character.json inside constants to match. One action has info and the other is null.

**Review by Apexal — APPROVED** [review]

**Pull request merged**